paddle/operators/fill_zeros_like_op.cu

```diff
@@ -0,0 +1,6 @@
+#include "paddle/framework/op_registry.h"
+#include "paddle/operators/fill_zeros_like_op.h"
+
+REGISTER_OP_GPU_KERNEL(
+    fill_zeros_like,
```

Comment:
`zeros_like` is not a verb. But all operator should be a `verb`. I think fill_zeros_like is more explicit.

Also, `zeros_like` may be implemented without filling anything into memory but a flag. Here named `fill` indicate that actually a new memory is allocated.